src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -0,0 +1,1503 @@
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+               <xbrli:context id="Director18_CY">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="uk-bus:EntityOfficersDimension">uk-bus:Director18</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:startDate>{{$current_period_start_on}}</xbrli:startDate>
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+               <xbrli:context id="Director19_CY">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="uk-bus:EntityOfficersDimension">uk-bus:Director19</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:startDate>{{$current_period_start_on}}</xbrli:startDate>
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+               <xbrli:context id="Director20_CY">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="uk-bus:EntityOfficersDimension">uk-bus:Director20</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:startDate>{{$current_period_start_on}}</xbrli:startDate>
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+              <xbrli:context id="CompanySecretary_CY">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="uk-bus:EntityOfficersDimension">uk-bus:CompanySecretary1</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:startDate>{{$current_period_start_on}}</xbrli:startDate>
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+              <xbrli:context id="Debtors_CY_END">
```

Comment:
Debtors current/previous can be removed as out of scope.

Reply:
debtors is still in mvp so is it worth keeping in?

Reply:
Sorry, didn't realise what was/wasn't MVP just thought that we didn't require things such as debtors until they are implemented. May be worth checking with @jsimons1989 to what the best approach is.

Reply:
What's merged into develop should be releasable. This would mean a document would be generated with the debtors iXBRL context but no matching debtors data in the document?

Reply:
Correct, the context will remain though the debtors data will not be present.